crates/holochain_types/src/web_app/web_app_manifest.rs

```diff
@@ -96,9 +99,10 @@ impl WebAppManifest {
     /// Get the location of the app bundle included in the manifest
     pub fn happ_bundle_location(&self) -> ResourceIdentifier {
         match self {
-            Self::V1(WebAppManifestV1 { happ_manifest, .. }) => {
-                resource_id_for_happ(&happ_manifest.path)
-            }
+            Self::V1(WebAppManifestV1 {
+                happ: happ_manifest,
+                ..
+            }) => resource_id_for_happ(&happ_manifest.path),
```

Comment:
```suggestion
                happ,
                ..
            }) => resource_id_for_happ(&happ.path),
```

Probably needs formatting after this change.